src/transforms/babel/babelrc.js

```diff
@@ -0,0 +1,219 @@
+const {buildRootChain} = require('@babel/core/lib/config/config-chain');
```

Comment:
These are not public APIs. Doing this means your code could break at any time.

Reply:
And that sucks, because when people see errors about this being a missing file, they'll come yell at us, not you.

Reply:
oops, I was experimenting. Those imports aren't used.

Reply:
Ahh alright sorry, I've seen real code doing this in other projects and it's endlessly frustrating.